src/Components/SignInForm/SignInForm.module.scss

```diff
@@ -0,0 +1,157 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap");
+
```

Comment:
Import the font in index.scss and create a css variable and then use it in required files